src/system.yml

```diff
@@ -12,18 +12,6 @@ esmodules:
 styles:
   - swnr.css
 packs:
-  - name: foci
-    label: Foci
-    system: swnr
-    module: swnr
-    path: packs/foci.db
-    entity: Item
-  - name: ranged
-    label: Ranged Weapons
-    system: swnr
-    module: swnr
-    path: packs/ranged.db
-    entity: Item
```

Comment:
Reordering a list is something that should be done in another PR.

Reply:
Done in #36.